Modules/Sources/JetpackStats/Cards/ChartCard.swift

```diff
@@ -77,31 +72,13 @@ struct ChartCard: View {
             StatsCardTitleView(title: metric.localizedTitle, showChevron: false)
             Spacer(minLength: 0)
         }
-        .animation(.spring, value: backButtonTitle)
         .accessibilityElement(children: .combine)
         .accessibilityLabel(Strings.Accessibility.cardTitle(metric.localizedTitle))
     }
 
-    @ViewBuilder
-    private var backButton: some View {
-        if let title = backButtonTitle, let action = backButtonAction {
-            Button(action: action) {
-                HStack(spacing: 4) {
-                    Image(systemName: "chevron.backward")
-                        .font(.system(size: 12, weight: .bold))
-                    Text(title)
-                        .font(.system(size: 14, weight: .medium))
-                }
-                .foregroundColor(Constants.Colors.blue)
-            }
-            .buttonStyle(.plain)
-            .transition(.scale(scale: 0.9).combined(with: .opacity))
-        }
-    }
-
     @ViewBuilder
     private var contentView: some View {
-        VStack(spacing: Constants.step0_5) {
+        VStack(spacing: 4) {
```

Comment:
There are a couple of similar minor tweaks to make the screen a bit tighter, so that there is space for the "Today" card, the "Chart", and the top of the "Most Viewed" card.